shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/user/UserService.kt

```diff
@@ -0,0 +1,27 @@
+package co.nimblehq.mark.kmmic.data.service.user
+
+import co.nimblehq.mark.kmmic.data.service.api.ApiService
+import co.nimblehq.mark.kmmic.data.service.api.extension.path
+import co.nimblehq.mark.kmmic.data.service.user.model.UserResponse
+import io.ktor.client.request.HttpRequestBuilder
+import io.ktor.http.HttpMethod
+import kotlinx.coroutines.flow.Flow
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+internal interface UserService {
+    fun me(): Flow<UserResponse>
```

Comment:
Maybe improve the naming?
```suggestion
    fun getProfile(): Flow<UserResponse>
```

Reply:
@suho updated at [3225037](https://github.com/markgravity/kmm-ic/pull/72/commits/3225037d11388c90ad505fdf1de85c0ac1d7d6f2)